agent/js/src/wd_server.js

```diff
@@ -1142,8 +1160,11 @@ WebDriverServer.prototype.runPageLoad_ = function(browserCaps) {
     this.startChrome_(browserCaps);

```

Comment:
this doesn't seem to be a problem in practice even though I think it should... Does Chrome always get closed in between runs?